src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -214,6 +199,36 @@ where
                     continue 'send_task;
                 }
             }
+
+            // 2.3 Send the compaction task.
+            if let Err(e) = compactor
+                .send_task(Task::CompactTask(compact_task.clone()))
+                .await
+            {
+                tracing::warn!(
+                    "Failed to send task {} to {}. {:#?}",
+                    compact_task.task_id,
+                    compactor.context_id(),
+                    e
+                );
+                // Cancel the task at best effort
+                compact_task.task_status = false;
```

Comment:
`cancle_compaction` logic similar with `need_cancel_task` in `hummock/manager/mod.rs`, shall we provide  a unified function ?

Rest LGTM ！ thanks

Reply:
Agree. But let me keep it straightforward for now, to reduce potential conflict with another RP (reliable task cancellation).